libraries/aws_cloudfront_distribution.rb

```diff
@@ -70,6 +70,19 @@ def initialize(opts = {})
     @custom_origin_ssl_protocols = @custom_origin_ssl_protocols.uniq.sort
   end
 
+  def s3_origin_path
+    catch_aws_errors do
+      @resp_config = @aws.cloudfront_client.get_distribution(id: opts[:distribution_id])
+    end
+    return if @resp_config.nil?
+
+    config = @resp_config.distribution.distribution_config
+
+    # AWS CloudFront web distribution origin path
+    # Either blank , or return path
+    @origin_path = config.origins.items.first['origin_path']
```

Comment:
the origin path could be different for different origins than why pick only first path?
Also no point of assigning it to an instance variable !